### PR TITLE
WIP: CI vs RPM build

### PR DESCRIPTION
**Pull request by amper43**
added:
- rpm spec
- dockerfile for rpmbuilder container
- hasic service
- makefile for CI

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.PHONY: rpm deb
+
+all: rpm deb
+
+rpm:
+				docker run --rm -v `pwd`/server:/root/rpmbuild/server \
```
Comment (jonyrock) [inline Makefile]: you know about this indention

File: build/rpm_build/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM rpmbuild/centos7
+
+USER root
+
+WORKDIR /root/rpmbuild
+
+RUN     yum -y install https://centos7.iuscommunity.org/ius-release.rpm epel-release
+RUN     yum -y install python36 python36-devel python36-setuptools gcc-c++ make \
+        &&  easy_install-3.6 pip \
+        &&  curl -sL https://rpm.nodesource.com/setup_8.x | bash - \
```
Comment (jonyrock) [inline build/rpm_build/Dockerfile]: can we move it to next `RUN`

File: build/rpm_build/hastic-server.spec [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Name: %{name}
+Version: %{version}
+Release: %{release}
+Summary: hastic-server
+
+Group: Installation Script
+License: Apache-2.0
+URL: hastic.io
+BuildRoot: %{buildroot}
+Requires: nodejs >= 6
```
Comment (jonyrock) [inline build/rpm_build/hastic-server.spec]: can we put here `nodejs >= 6.14` ?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+rpm_node6:
+	docker run --rm -v `pwd`/server:/root/rpmbuild/server \
+			-v `pwd`/analytics:/root/rpmbuild/analytics \
+			-v `pwd`/build/rpmbuild:/root/rpmbuild/rpm \
+			-v `pwd`/dist/RPMS_6:/root/rpmbuild/RPMS \
+			-e "NODE_VERSION=6.14.0" \
+			-e "RPM_NODE_VERSION=6" \
+			-e "HASTIC_RELEASE_VERSION=`cat server/package.json| jq -r .version | sed 's/-/_/g'`" \
+			amper43/hastic-rpmbuilder rpmbuild -ba rpm/hastic-server.spec
```
Comment (rozetko) [inline Makefile]: @jonyrock we need to create a dockerhub repo for it: `hastic/rpmbuilder`

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+			-e "NODE_VERSION=8.0.0" \
+			-e "RPM_NODE_VERSION=8" \
+			-e "HASTIC_RELEASE_VERSION=`cat server/package.json| jq -r .version | sed 's/-/_/g'`" \
+			amper43/hastic-rpmbuilder rpmbuild -ba rpm/hastic-server.spec
+
+rpm_node6:
+	docker run --rm -v `pwd`/server:/root/rpmbuild/server \
+			-v `pwd`/analytics:/root/rpmbuild/analytics \
+			-v `pwd`/build/rpmbuild:/root/rpmbuild/rpm \
+			-v `pwd`/dist/RPMS_6:/root/rpmbuild/RPMS \
```
Comment (jonyrock) [inline Makefile]: lets mode `dist` to `packaging` like in Grafana source code: https://github.com/grafana/grafana/blob/4f601761dfb85499c3b566fd39143662eb25d206/.gitignore#L61

File: Makefile (hunk 4 above)
Comment (jonyrock) [reply Makefile]: @rozetko do we really need this? why can`t we create a docker file on this repo?

File: build/rpmbuild/Makefile [hunk 6]
```diff
@@ -0,0 +1,3 @@
+all:
+	docker build  -t amper43/hastic-rpmbuilder .
+	docker push amper43/hastic-rpmbuilder
```
Comment (jonyrock) [inline build/rpmbuild/Makefile]: please stop using apmer43

File: Makefile (hunk 4 above)
Comment (rozetko) [reply Makefile]: @jonyrock I don't know if it's a good idea to build the same image every time we make a release

File: Makefile [hunk 7]
```diff
@@ -3,14 +3,14 @@
...
-	docker run --rm -v `pwd`/server:/root/rpmbuild/server \
+	docker run --rm -it -v `pwd`/server:/root/rpmbuild/server \
 			-v `pwd`/analytics:/root/rpmbuild/analytics \
 			-v `pwd`/build/rpmbuild:/root/rpmbuild/rpm \
 			-v `pwd`/dist/RPMS_8:/root/rpmbuild/RPMS \
 			-e "NODE_VERSION=v8.0.0" \
 			-e "RPM_NODE_VERSION=8" \
 			-e "HASTIC_RELEASE_VERSION=`cat server/package.json| jq -r .version | sed 's/-/_/g'`" \
-			amper43/hastic-rpmbuilder rpmbuild -ba rpm/hastic-server.spec
+			amper43/hastic-rpmbuilder bash #rpmbuild -ba rpm/hastic-server.spec
```
Comment (jonyrock) [inline Makefile]: amper ?

**Pull request merged**